docs/sql/oracle/onboarding/create-schema.sql

```diff
@@ -51,6 +51,7 @@ CREATE TABLE ES_ONBOARDING_OTP (
     ERROR_DETAIL VARCHAR2(256 CHAR),
     ERROR_ORIGIN VARCHAR2(256 CHAR),
     FAILED_ATTEMPTS INTEGER,
+    TOTAL_ATTEMPTS INTEGER DEFAULT 0,
```

Comment:
Sorry, one more thing. It should also be mentioned in `PowerAuth-Enrollment-Onboarding-Server-1.5.0.md`. I know, it is a little duplicity with Liquibase, but the update process is not fine-tuned yet.